app/adapters/application.js

```diff
@@ -1,9 +1,10 @@
 import RESTAdapter from '@ember-data/adapter/rest';
 import { inject as service } from '@ember/service';
 import { computed } from '@ember/object';
+import FastbootAdapter from 'ember-data-storefront/mixins/fastboot-adapter';
 
-export default RESTAdapter.extend({
-  fastboot: service(),
+export default RESTAdapter.extend(FastbootAdapter, {
+  fetcher: service(),
```

Comment:
Nah, I don't need fetcher but need and fastboot should be kept. Will fix.

Reply:
Does this comment still apply?

Reply:
No. I need both `fetcher` and `fastboot` now.